arches/app/models/models.py

```diff
@@ -907,15 +915,21 @@ def __init__(self, *args, **kwargs):
     def clean(self):
         if not self.alias:
             Graph.objects.get(pk=self.graph_id).create_node_alias(self)
+        self.grouping_node_id = self.nodegroup_id
```

Comment:
Based on our conversation yesterday I decided to run this unconditionally instead of when empty. That would make the changes in a9b2f32162e7c7b15206b9728658f95a392a0b88 not strictly necessary, but I left them in. (They'll probably change anyway after #11609.)